browser/ui/views/toolbar/brave_toolbar_view.cc

```diff
@@ -108,6 +108,8 @@ void BraveToolbarView::Init() {
 
   DCHECK(location_bar_);
   AddChildViewAt(bookmark_, GetIndexOf(location_bar_));
+  bookmark_->UpdateImage();
+  brave_initialized_ = true;
```

Comment:
shouldn't this also get set to true here? https://github.com/brave/brave-core/pull/2829/commits/8a47d4851ace90241ca27b4da86e344a6d91f256#diff-3d72382b1c4fba0ded2fc0100530ff54R91

Reply:
Fixed.